examples/preprocessing/plot_muscle_detection.py

```diff
@@ -0,0 +1,84 @@
+"""
+===========================
+Annotate muscle artifacts
+===========================
+
+Muscle contractions produce high frequency activity that can mask brain signal
+of interest. Muscle artifacts can be produced when clenching the jaw,
+swallowing, or twitching a head muscle. Muscle artifacts are most notable
+in the range of 110-140 Hz.
+
+This example uses :func:`~mne.preprocessing.annotate_muscle_zscore` to annotate
+segments where muscle activity is likely present. This is done by band-pass
+filtering the data in the 110-140 Hz range. Then, the envelope is taken using
+the hilbert analytical signal to only consider the absolute amplitude and not
+the phase of the high frequency signal. The envelope is z-scored and summed
+across channels and divedid by the square root of the channel numer. To remove
+noisy transient peaks (muscle artifacts last several hundred milliseconds),
+the channel averaged z-scored is low-pass filtered to 4 Hz. Segments above a
+set threshold are annotated as ``BAD_muscle``. In addition, ``min_length_good``
+parameter determines the cutoff for whether short spans of "good data" in
+between muscle artifacts are included in the surrounding "BAD" annotation.
+
+"""
+# Authors: Adonay Nunes <adonay.s.nunes@gmail.com>
+#          Luke Bloy <luke.bloy@gmail.com>
+# License: BSD (3-clause)
+
+import os.path as op
+import matplotlib.pyplot as plt
+import numpy as np
+from mne import pick_types
+from mne.datasets.brainstorm import bst_auditory
+from mne.io import read_raw_ctf
+from mne.preprocessing import annotate_muscle_zscore
+
+
+# Load data
+data_path = bst_auditory.data_path()
+data_path_MEG = op.join(data_path, 'MEG')
+subject = 'bst_auditory'
+raw_fname = op.join(data_path_MEG, subject, 'S01_AEF_20131218_01.ds')
```

Comment:
```suggestion
raw_fname = op.join(data_path, 'MEG', 'bst_auditory', 'S01_AEF_20131218_01.ds')
```
both `data_path_MEG` and `subject` are only used once, and it's not obvious to me that the variable names you assign add any clarity to the code, so I'd omit them here and put the values directly into the final `op.join` call.